.spin/cmds.py

```diff
@@ -9,9 +9,21 @@
 
 @click.command()
 @click.option(
-    "--clean", is_flag=True, help="Clean previously built docs before building"
+    "--clean", is_flag=True,
+    default=False,
+    help="Clean previously built docs before building"
 )
-def docs(clean=False):
+@click.option(
+    "--raise-on-warning/--no-raise-on-warning",
+    default=True,
+    help="Convert warnings to errors"
+)
+@click.option(
+    "--install-deps/--no-install-deps",
+    default=True,
```

Comment:
```suggestion
    default=True,
    envvar="SPIN_AUTOINSTALL_DOC_DEPS",
```
Might be useful for those who want to change the default behavior locally...